src/sentry/models/featureadoption.py

```diff
@@ -0,0 +1,149 @@
+    Feature("breadcrumbs", "Breadcrumbs", "code", prerequisite=["first_event", ("python", "javascript", "node", "php")]),
+    # Feature("resolve_in_commit", "Resolve in Commit", "code", prerequisite=["first_event", "releases"]),
+
+    # Web UI
+    Feature("first_project", "First Project", "web"),
+    Feature("invite_team", "Invite Team", "web", prerequisite=["first_project"]),
+    Feature("assignment", "Assign Issue", "web", prerequisite=["invite_team", "first_event"]),
+    Feature("resolved_in_release", "Resolve in Next Release", "web", prerequisite=["release_tracking"]),
+    # Feature("snooze", "Snooze Issue", "web", prerequisite=["first_event"]),
+    # Feature("merge", "Merge Issues", "web", prerequisite=["first_event"]),
+    Feature("advanced_search", "Advanced Search", "web", prerequisite=["first_event"]),
+    Feature("saved_search", "Saved Search", "web", prerequisite=["advanced_search"]),
+    Feature("inbound_filters", "Inbound Filters", "web", prerequisite=["first_event"]),
+    Feature("alert_rules", "Alert Rules", "web", prerequisite=["first_event"]),
+    Feature("issue_tracker_integration", "Issue Tracker Integration", "web", prerequisite=["first_project"]),
+    Feature("notification_integration", "Notification Integration", "web", prerequisite=["first_project"]),
+    # Feature("releases", "Releases", "web", prerequisite=["first_project"]),
+
+    # Admin UI
+    Feature("sso", "SSO", "admin", prerequisite=["invite_team"]),
+    Feature("data_scrubbers", "Data Scrubbers", "admin", prerequisite=["first_event"]),
+
+    # Future features
+    # Feature("Health Metrics", "code"),
+    # Feature("Customer Support Integration", "web", prerequisite=["User Tracking"]),
+    # Feature("ETL Integration", "web", prerequesite=["First Event"]),
+    # Feature("2-Factor Auth", "web", prerequisite=["Invite Team"]),
+]
+
+# No feature slug should be duplicate
+assert len(features) == len(set([f.slug for f in features]))
+
+
+class FeatureAdoptionManager(BaseManager):
+    def record(self, organization_id, feature_slug, **kwargs):
+        cache_key = 'featureadoption:%s:%s' % (
+            organization_id,
+            feature_slug,
+        )
+        if cache.get(cache_key) is None:
+            row, created = self.create_or_update(
+                organization_id=organization_id,
+                feature_slug=feature_slug,
+                values={
+                    'date_modified': timezone.now(),
+                    'complete': True,
+                },
+                defaults={
+                    'applicable': True,  # Only on the first time should override CS
+                },
+            )
+            # Store marker to prevent running all the time
+            cache.set(cache_key, 1, 3600)
+            return created
+
+        return False
+
+
+class FeatureAdoption(Model):
+    __core__ = False
+
+    organization = FlexibleForeignKey('sentry.Organization')
+    feature_slug = models.SlugField()
```

Comment:
Definitely don't need `SlugField` here. And I'd prefer this isn't a free-form text field. It should just be a lightweight integer and something that `Feature` objects should support.